inc/AccessToken.php

```diff
@@ -46,15 +46,31 @@ public static function get_token() {
 		return (string) $token;
 	}
 
+	/**
+	 * Get the stored user_id, if available.
+	 *
+	 * @return string User_id or an empty string if not available.
+	 */
+	public static function get_user() {
+		$user       = '';
+		$expiration = get_option( 'bluehost_access_token_expiration', 0 );
```

Comment:
Calling this out in line: I wonder if checking expiration is helpful here. I know it'd be more simple to just return the user here, but should we check that the token isn't expired to ensure that the user_id will be useful? Probably overthinking it here. Thoughts?

Reply:
@circlecube I don't think we NEED to expire the user, but for an initial release, it is probably a good idea. Until we've figured out how to support multiple users, let's just keep the two in sync.